src/app/sidenav/sidenav.component.ts

```diff
@@ -11,8 +11,9 @@ import { faHeart } from '@fortawesome/free-regular-svg-icons';
 export class SidenavComponent implements OnInit {
   public image: string = "";
   public rss: string;
-	
+
 	public faFire = faFire;
+  public faBolt = faBolt;
```

Comment:
Indentation is a bit weird.
I've an ambition to create a styleguide sometime in the future but no ETA for that yet.